includes/scripts.php

```diff
@@ -353,6 +353,7 @@ function edd_register_admin_styles() {
 
 	// Register styles
 	wp_register_style( 'jquery-chosen',         $css_dir . 'chosen'               . $css_suffix, array(), $version );
+	wp_register_style( 'jquery-ui-fresh',       $css_dir . 'jquery-ui-fresh'      . $css_suffix, array(), $version );
```

Comment:
🤦 Sorry @LisaCee, these diffs are messing with my brain today. 

The script handle (first argument) should still be `jquery-ui-css`, and then `jquery-ui-fresh` is used to reference the file names: `jquery-ui-fresh.css` and `jquery-ui-fresh.min.css`.

Sorry for the back and forth.